extra/3-magic-8-ball.js

```diff
@@ -82,12 +136,12 @@ test("whole magic 8 ball sequence", () => {
   expect(consoleLogSpy).toHaveBeenCalledTimes(1);
   expect(consoleLogSpy).toHaveBeenLastCalledWith("The ball has shaken!");
 
-  expect(checkAnswer(answer)).toBeOneOf([
-    "very positive",
-    "positive",
-    "negative",
-    "very negative",
-  ]);
+  // expect(checkAnswer(answer)).toBeOneOf([
+  //   "very positive",
+  //   "positive",
+  //   "negative",
+  //   "very negative",
+  // ]);
```

Comment:
I can see that this test is not working - I get "TypeError: expect(...).toBeOneOf is not a function", which I think is a problem with the jest-extended library. Maybe bring this up on Slack or at the next Saturday class to see if anyone know how to solve it.

Reply:
thank you Karen